src/browser/styles/app.scss

```diff
@@ -4,117 +4,30 @@ body {
     font-family: 'Roboto', sans-serif;
 }
 
-.nav-item {
-    padding: .5em 1em;
-}
-
-.user {
-    float: right;
-    padding: 1em 2em;
-}
-
-.event-view-container {
-    margin: 0 2em;
-}
-
-
-.event-view-name {
-    font-size: 20px;
-    font-weight: 400;
-    margin: 29px 0 43px 0;
-}
-
-.event-view-address, .event-view-date {
-    display: inline-block;
-    font-size: 12px;
-    font-weight: 400;
-    opacity: .54;
-}
-
-.event-view-address {
-    margin: 0 0 14px 4px;
-}
-
-.event-view-date {
-    margin: 0 0 38px 4px;
-}
-
-.event-view-description {
-    font-size: 14px;
-    color: #9E9E9E;
-    font-weight: 400;
-    margin: 0;
-}
-
-
-
-.event-headers {
-    float: left;
```

Comment:
I'm usually against floats in these situations. If it's concerning texts, do inline-blocks. If it's layouts, floats are alright. In this case, were doing text :)